.pre-commit-config.yaml

```diff
@@ -6,7 +6,7 @@ repos:
         exclude: conda/meta.yaml
  
   - repo: https://github.com/psf/black
-    rev: 22.6.0
+    rev: 23.1.0
```

Comment:
Thanks. Please also update `pre-commit-hooks` to `v4.4.0` and `isort` to `5.12.0`.

Reply:
Done in the commit: [af66f0b](https://github.com/lsst-ts/ts_m2com/pull/26/commits/af66f0bb11ebcfc7f66ecdff70f7470a6c826a94). Thanks!